aeron-archive/src/main/java/io/aeron/archive/ArchiveConductor.java

```diff
@@ -132,6 +136,15 @@ abstract class ArchiveConductor
         dutyCycleTracker = ctx.conductorDutyCycleTracker();
         cachedEpochClock.update(epochClock.time());
 
+        try
+        {
+            secureRandom = SecureRandom.getInstanceStrong();
```

Comment:
Beware of using getInstanceStrong() : it can block for several seconds on some systems and deplete the entropy.

Not sure if there is a need for strong security here or if this is simply following the recommendation of some static analysis tool. I've been bitten in the past by doing this :)

Reply:
It is being used to generate an authentication token, so I've picked the strongest generator as a conservative option.

Reply:
I've moved the `getInstanceStrong()` usage to be there to construct a seed for the PRNG.